### ISSUE TITLE
The icon display of ibus-mozc has an issue.

### ISSUE THREAD

**Issue by TDYQ-Liu** [google/mozc#756]
**Description**
The icon display of ibus-mozc has an issue. The character encoding "3042" should be displayed as "あ".

**Steps to reproduce**

Upgrading from Fedora 37 to 38. Executed the following commands according to the Fedora documentation.

```sh
sudo dnf system-upgrade download --releasever=38
sudo dnf system-upgrade reboot
```

**Expected behavior**

Make the encoding display correctly.

**Actual behavior**
A clear and concise description of what you actually got.

Restart the system. 
Delete the configuration file of Mozc and reinstall ibus-mozc.
Run ibus exit and ibus-daemon --xim & , i can reproduce my problem
Run ibus-daemon --xim --verbose & and look at the output no errors occurred.

**Screenshots**

![Screenshot from 2023-06-05 18-48-46](https://github.com/google/mozc/assets/71477981/b24b5cfb-81ba-4c8c-b8c0-4e3b62243948)
![Screenshot from 2023-06-05 19-46-02](https://github.com/google/mozc/assets/71477981/b6eaacf6-da52-408a-83b9-48be5e3b76d1)

**Version or commit-id**
[e.g. Mozc-2.28.4960.100+24.11.oss or d50a8b9ae28c4fba265f734b38bc5ae392fe4d25]
You can get the version string by converting "Version" or "ばーじょん".

Name         : ibus-mozc
Version      : 2.28.4950.102
Release      : 3.fc38
Architecture : x86_64
Source       : mozc-2.28.4950.102-3.fc38.src.rpm

**Environment**
 - OS: **fedora 38**
 - IMF (for Linux): **Ibus**
 - Related Applications **gnome 44.1**

**Investigations**
 - Whether this issue happens on both Ibus and other IMF (e.g. Fcitx, uim).
   + yes, no, **N/A**
 - Whether this issue happens on other IMEs (e.g. Anthy, SKK).
   + yes, no, **N/A**
 - What applications this issue happens on (e.g. Chromium, gedit).
   + list of applications
 - What applications this issue does not happen on (e.g. Chromium, gedit).
   + list of applications, or **not-found**
 - (optional) What versions or commit-ids this issue did not happen on
   (e.g. Mozc-2.28.4960.100+24.11.oss).
   + list of versions or commit-ids, or no-investigation.
**The version of Fedora 37.**

**Additional context**

https://github.com/ibus/ibus/issues/2517#issuecomment-1576836026

```sh
# run /usr/libexec/ibus-engine-mozc --xml
<engines>
<engine>
  <description>Mozc (Japanese Input Method)</description>
  <language>ja</language>
  <icon>/usr/share/ibus-mozc/product_icon.png</icon>
  <rank>80</rank>
  <icon_prop_key>InputMode</icon_prop_key>
  <symbol>あ</symbol>
  <setup>/usr/libexec/mozc/mozc_tool --mode=config_dialog</setup>
  <name>mozc-jp</name>
  <longname>Mozc</longname>
  <layout>default</layout>
  <layout_variant></layout_variant>
  <layout_option></layout_option>
</engine>
</engines>

# run /usr/libexec/ibus-engine-anthy --xml
<?xml version="1.0" encoding="utf-8"?>
<engines>
	<engine>
		<name>anthy</name>
		<language>ja</language>
		<license>GPL</license>
		<author>Peng Huang &lt;shawn.p.huang@gmail.com&gt;</author>
		<icon>ibus-anthy</icon>
		<layout>default</layout>
		<layout_variant></layout_variant>
		<layout_option></layout_option>
		<longname translatable="no">Anthy</longname>
		<description>Anthy Input Method</description>
		<rank>99</rank>
		<hotkeys>Control+space,Zenkaku_Hankaku</hotkeys>
		<symbol>&#x3042;</symbol>
		<icon_prop_key>InputMode</icon_prop_key>
		<version>1.5.14</version>
		<textdomain>ibus-anthy</textdomain>
	</engine>
</engines>
```

**Comment by hiroyuki-komatsu** [comment]
Hi TDYQ-Liu,

Thank you for your report.
We hope 5ca52be fixes the issue.

Would you try it on your environment?
To apply the change, you need to remove the existing ibus_config.textproto in ~/.confing/mozc/ or ~/.mozc/.

Best,

**Comment by TDYQ-Liu** [comment]
> Hi TDYQ-Liu,
> 
> Thank you for your report. We hope [5ca52be](https://github.com/google/mozc/commit/5ca52be64d4d50ac123a3c2eff5834eae308f39b) fixes the issue.
> 
> Would you try it on your environment? To apply the change, you need to remove the existing ibus_config.textproto in ~/.confing/mozc/ or ~/.mozc/.
> 
> Best,

May I ask how can I apply this modification? The Fedora repository hasn't been updated.
https://packages.fedoraproject.org/pkgs/mozc/ibus-mozc/

**Comment by TDYQ-Liu** [comment]
![Screenshot from 2023-06-07 09-26-26](https://github.com/google/mozc/assets/71477981/2cb7b563-f5d9-40eb-b4bc-9074d85eb39f)

**Comment by TDYQ-Liu** [comment]
The error has been fixed in *[ibus-mozc-2.28.4950.102-6.fc38](https://koji.fedoraproject.org/koji/buildinfo?buildID=2223999)*.
